### ISSUE TITLE
Deprecation warnings first instead of removing method and Bug

### ISSUE THREAD

**Issue by Stradivario** [krzkaczor/TypeChain#92]
Hello there,

I just installed latest typechain version globally and all of my build processes failed!

What is happening is that '--force' argument is deprecated.
Added some --target property for Truffle, web3, legacy

By now i was using legacy and now when i run it it says:

```
> typechain --target=legacy --outDir src/app/core/contracts './contracts/build/contracts/*.json'

Error occured:  ENOENT: no such file or directory, open '/home/rampage/.nvm/versions/node/v8.9.4/lib/node_modules/typechain/dist/targets/legacy/runtime/typechain-runtime.ts'
```

I am sure this is not intended ?

So basically i don't have much time to waste and will downgrade `typechain` to older version.

Please write some CHANGELOG or somewhere inside the REPO what is changed for example it is important if you decide to remove --force to write it somewhere.Thanks

Regards,
K.T.

**Comment by Stradivario** [comment]
Also when i try to install it globally with NODE 10.^ it is not installing with the same error.

**Comment by krzkaczor** [comment]
Thanks for the report. Fix was deployed as `typechain@0.3.4`, I also added tests that should cover it in future. I would appreciate note if this solved your problems.

> What is happening is that '--force' argument is deprecated.

Typechain 0.3.x is based on `ts-generator` project. I decided that having `--force` is not a good option since it should be default settings (and you should never commit generated files etc.). Implementing it in `ts-generator` would require additional work so I just skipped it. I know that it's a breaking change but very minor one and we still didnt hit 1.0 so it's expected IMO that things like this could happen.

> Please write some CHANGELOG or somewhere inside the REPO what is changed for example it is important if you decide to remove --force to write it somewhere.Thanks

It's already there :) https://github.com/krzkaczor/TypeChain/blob/master/CHANGELOG.md

I hope that next time updating Typechain will be a smoother process.

**Comment by Stradivario** [comment]
@krzkaczor awesome man!!! I will check more carefully next time !!
Thank you very much!
